usim_pytest/test_types/test_pipe.py

```diff
@@ -0,0 +1,60 @@
+from usim import time, Pipe, Scope
+
+from ..utility import via_usim
+
+
+async def aenumerate(aiterable, start=0):
+    count = start
+    async for item in aiterable:
+        yield count, item
+        count += 1
+
+
+class TestPipe:
```

Comment:
Could you please also add a unit test for testing your assertions? So entering ``0`` and negative values?